### ISSUE TITLE
How to build and access with `loop_unroller` and `opt_loops` binaries in CI

### ISSUE THREAD

**Issue by mostafaelhoushi** [facebookresearch/CompilerGym#707]
I think that we should do the following:
- The `loop_optimizations_service` example environment should hopefully become a full environment. So I think we can add `opt_loops` binary to the .whl file. If that's the case, should we move `loop_optimizations_service` from `examples` directory to `envs` directory?
- The `example_unrolling_service` is meant to be a small example to guide users on how to create their own environment. Ideally, the `loop_unroller` should be built using a standalone script without having to be part of the repo's CMake or bazel build. I already started doing that in a PR (but then I undid it in this [commit](https://github.com/facebookresearch/CompilerGym/pull/642/commits/a79e4cf163af437c05cc7010d76a3d6d3230f21f)), but will need to redo it properly and clean things up.

**Comment by ChrisCummins** [comment]
> * The `loop_optimizations_service` example environment should hopefully become a full environment. So I think we can add `opt_loops` binary to the .whl file. If that's the case, should we move `loop_optimizations_service` from `examples` directory to `envs` directory?

I agree. It'll need a reference page, like [this](https://compilergym.com/llvm/index.html)
